nodebuilder/tests/swamp/swamp_tx.go

```diff
@@ -10,25 +10,35 @@ import (
 )
 
 // FillBlocks produces the given amount of contiguous blocks with customizable size.
-// The returned channel reports when the process is finished.
-func FillBlocks(ctx context.Context, cctx testnode.Context, account string, bsize, blocks int) chan error {
+// The returned channels report the submitted heights and when the process is finished.
+func FillBlocks(
+	ctx context.Context,
+	cctx testnode.Context,
+	account string,
+	bsize, blocks int,
+) (
+	<-chan int64, <-chan error,
+) {
 	errCh := make(chan error)
```

Comment:
BTW, why we don't close this channel?

Reply:
dunno. will close

Reply:
also...I think it should be buffered. With size 1.

Reply:
makes sense